packages/flutter/lib/src/rendering/sliver_grid.dart

```diff
@@ -315,6 +317,9 @@ class SliverGridDelegateWithFixedCrossAxisCount extends SliverGridDelegate {
   /// The ratio of the cross-axis to the main-axis extent of each child.
   final double childAspectRatio;
 
+  /// The extent of each tile in the main axis.
```

Comment:
Here as well we should explain how this is used and what the default behavior is.